lib/js/components/Pagination/Pagination.stories.ts

```diff
@@ -0,0 +1,47 @@
+import Pagination from './Pagination.vue';
+import { PAGINATION_LAYOUTS } from './Pagination.consts';
+
+import { ArgTypes, Meta, StoryFn } from '@storybook/vue';
+
+export default {
+	title: 'Components/Pagination',
+	component: Pagination,
+} as Meta<typeof Pagination>;
+
+const StoryTemplate: StoryFn<typeof Pagination> = (argTypes, { updateArgs }) => ({
+	components: { Pagination },
+	props: Object.keys(argTypes),
+	methods: {
+		onChangePage(currentPage) {
+			updateArgs({ currentPage });
+		},
+	},
+	template: `
+		<Pagination v-bind=$props
+					@change-page='onChangePage'>
+		<div v-if='accessory' slot='accessory' v-html='accessory' />
```

Comment:
Let's use the new syntax slot `#accessory`. I think it'll require a `template` wrapper.